apps/web/modules/ee/survey-follow-ups/components/FollowUpModal.tsx

```
@@ -0,0 +1,702 @@
+                                      if (!selectedOption) return;
+
+                                      field.onChange(selectedOption.id);
+                                    }}>
+                                    <SelectTrigger className="overflow-hidden text-ellipsis whitespace-nowrap">
+                                      <SelectValue />
+                                    </SelectTrigger>
+
+                                    <SelectContent>
+                                      {emailSendToOptions.map((option) => {
+                                        return (
+                                          <SelectItem value={option.id}>
```

Biome

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)